src/app.js

```diff
@@ -10,12 +10,13 @@ const loadConfig = require('./lib/config')
 const RoutingTables = require('./lib/routing-tables')
 const RouteBuilder = require('./lib/route-builder')
 const RouteBroadcaster = require('./lib/route-broadcaster')
+const CurveCache = require('./lib/curve-cache')
```

Comment:
Okay, so in that case sometimes a node may want to remember what it has promised, but I'm not sure whether we should base ilp-kit's architecture on the need to support that.
Maybe to simplify this question; should liquiditycurves be stored:
a) where they are now, in the routing table (which is PrefixMap)
b) in the newly proposed CurveCache (choice of nextHop would stay in the routing table)
c) spread across both these two places, depending on how the liquidity curve was obtained